CHANGELOG.md

```diff
@@ -2,6 +2,85 @@
+- The browser bundle now relies on the native Web Crypto API instead of CryptoJS. The `ably/build/ably.noencryption` bundle has been removed, as it is no longer necessary.
+- The separate bundle for web workers, `ably/build/ably-webworker`, has been removed. You can now import from `ably` in web workers.
+- The Types namespace has been removed. All types are now exported at the top level.
+
+Read more in [the migration guide for v2](link TBD).
+
+BREAKING CHANGES
+
+- Remove code that's supporting older platforms [\#1629](https://github.com/ably/ably-js/pull/1629)
+- `ChannelProvider` implementation [\#1620](https://github.com/ably/ably-js/pull/1620)
+- Remove duplicate type names and change public `ChannelModes` type [\#1601](https://github.com/ably/ably-js/pull/1601)
+- Remove `any` from `stats()` param type [\#1561](https://github.com/ably/ably-js/pull/1561)
+- Make modular variant of the library available to Web Workers [\#1550](https://github.com/ably/ably-js/pull/1550)
+- Remove false class exports in type declarations [\#1524](https://github.com/ably/ably-js/pull/1524)
+- Implement untyped stats API [\#1522](https://github.com/ably/ably-js/pull/1522)
+- Remove the `Types` namespace [\#1518](https://github.com/ably/ably-js/pull/1518)
+- Update publishing methods to accept a `Message`-shaped object [\#1515](https://github.com/ably/ably-js/pull/1515)
+- Remove the existing plugins mechanism [\#1512](https://github.com/ably/ably-js/pull/1512)
+- Remove `noencryption` variant of the library [\#1500](https://github.com/ably/ably-js/pull/1500)
+- Remove public callbacks API [\#1358](https://github.com/ably/ably-js/pull/1358)
+- Fix the type definition of `Crypto.getDefaultParams` [\#1352](https://github.com/ably/ably-js/pull/1352)
+- Change `Crypto.generateRandomKey` API to use Promises [\#1351](https://github.com/ably/ably-js/pull/1351)
+- Replace CryptoJS library with Web Crypto API in web bundle [\#1333](https://github.com/ably/ably-js/pull/1333), [\#1325](https://github.com/ably/ably-js/pull/1325), [\#1299](https://github.com/ably/ably-js/pull/1299)
+- Describe the types of key returned by `generateRandomKey` [\#1320](https://github.com/ably/ably-js/pull/1320)
+- Make `ICipher.decrypt` async [\#1311](https://github.com/ably/ably-js/pull/1311)
+- Add mandatory `version` param to `Rest.request` [\#1231](https://github.com/ably/ably-js/pull/1231)
+- Remove `ably-commonjs*.js` files [\#1229](https://github.com/ably/ably-js/pull/1229)
+- Remove remaining deprecated APIs [\#1227](https://github.com/ably/ably-js/pull/1227)
+- Remove deprecated `fromEncoded*` type declarations [\#1222](https://github.com/ably/ably-js/pull/1222)
+- Conform to spec for logging configuration [\#1216](https://github.com/ably/ably-js/pull/1216)
```

Comment:
I think we should avoid mentioning the spec in changelog entries whenever possible, feel free to editorialise some of the PR titles

Reply:
I assume this is because "the spec" term is not something that users are familiar with, and there is no point to expose them to it, right?
So a more user-friendly explanation on **what** actually was done would be much more useful.

Reply:
see https://github.com/ably/ably-js/pull/1656/commits/dd8db9409cb3cde90bfc321ddd33ea71bd3009c2